README.md

```diff
@@ -323,7 +323,8 @@ down --way --the --all --my-custom-flag
 
 The rough heuristic here is if we have custom arguments:
 
-1. If a `builder <action>` command, append with ` -- ` to pass through.
+1. If a `builder <action>` command, pass through using builder-specific
```

Comment:
This is just an update. Above I have https://github.com/FormidableLabs/builder/blob/bug-propagate-extra-args/README.md#custom-flags